server/consumer.go

```diff
@@ -459,7 +459,8 @@ func setConsumerConfigDefaults(config *ConsumerConfig, streamCfg *StreamConfig,
 	if config.MaxAckPending == 0 {
 		config.MaxAckPending = streamCfg.ConsumerLimits.MaxAckPending
 	}
-	if config.InactiveThreshold == 0 {
+    // Do not inherit inactivity threshold for durable consumers.
```

Comment:
needs go fmt here maybe?

Reply:
yeah, lsp died and did not format. Fixed :).